style.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url("https://user-images.githubusercontent.com/66992901/198218680-01a4bf79-49e0-4ca5-b5a4-363c351466a5.png");
+  background-repeat: no-repeat;
+}
+
```

Comment:
There are some few disparities between your design and that of the figma. Kindly check and fix the white/blank space on the right side of your design. Also check the fonts and spacing to make sure it matches the design.

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-28 06-11-21](https://user-images.githubusercontent.com/65233860/198515825-f509482b-0cbc-4783-89fb-21e7287a208d.png)  |  ![Screenshot from 2022-10-28 05-25-32](https://user-images.githubusercontent.com/65233860/198510369-78d07027-522b-416d-acff-2d29123d8dc4.png)